src/main/components/form/claimDetailsValidator.ts

```diff
@@ -0,0 +1,54 @@
+export const validatePersonalDetails = (userCase: Record<string, any>): boolean => {
+  if (!userCase) {
+    return false;
+  }
+  const { typeOfClaim, address1, addressTown, addressPostcode, addressCountry } = userCase;
+
+  return !(!typeOfClaim || !address1 || !addressTown || !addressPostcode || !addressCountry);
```

Comment:
typeOfClaim won't be in the personal details sections

```suggestion
  const {address1, addressTown, addressPostcode, addressCountry } = userCase;

  return !(!address1 || !addressTown || !addressPostcode || !addressCountry);
```

Reply:
yes. added this because it was on earlier triage questions. but since we are moving this, it's okay to remove